pom.xml

```diff
@@ -203,28 +203,51 @@
             <target>1.8</target>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-javadoc-plugin</artifactId>
+          <version>2.10.4</version>
+          <inherited>false</inherited>
+          <configuration>
+            <reportOutputDirectory>docs</reportOutputDirectory>
+            <destDir>apidocs</destDir>
+          </configuration>
+          <executions>
+            <execution>
+              <id>aggregate</id>
+              <phase>site</phase>
+              <goals>
+                <goal>aggregate</goal>
+              </goals>
+            </execution>
+          </executions>
+        </plugin>
+        <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>cobertura-maven-plugin</artifactId>
```

Comment:
I believe JaCoCo is generally preferred to Cobertura now days

Reply:
No problem with JaCoCo...will change it.
https://confluence.atlassian.com/clover/comparison-of-code-coverage-tools-681706101.html

Reply:
Thanks!!